app/components/emoji_picker/emoji_picker.js

```diff
@@ -317,6 +358,18 @@ export default class EmojiPicker extends PureComponent {
         this.sectionList = c;
     };
 
+    renderFooter = () => {
+        if (!this.state.missingPages) {
+            return null;
+        }
+
+        return (
+            <View style={{flex: 1, alignItems: 'center'}}>
```

Comment:
This should go in the StyleSheet

Reply:
I copied that from another component doing it this way but I'll throw it into the stylesheet if that's the proper way